moped-editor/src/views/projects/projectView/ProjectActivityLogDialog.js

```diff
@@ -107,10 +111,12 @@ const ProjectActivityLogDialog = ({ activity_id, handleClose }) => {
   };
 
   const getDateTime = data => {
-    const date = data?.moped_activity_log[0]?.created_at;
+    const timestampz = data?.moped_activity_log[0]?.created_at;
 
     try {
-      return new Date(date).toLocaleString('en-US', {timeZone: 'UTC'});
+      return `${formatTimeStampZType(timestampz)}, ${makeFullTimeFromTimeStampZ(
```

Comment:
Fixes time stamp in Activity Log dialog (details view) https://github.com/cityofaustin/atd-data-tech/issues/7121